bin/cli-meli-accounts/prompt.js

```diff
@@ -0,0 +1,36 @@
+const inquirer = require('inquirer')
+
+const choices = [
+  {
+    name: 'Create a new Test account',
+    value: 'newTestAccount',
+    short: 'Test account'
+  },
+  {
+    name: 'Login existing account',
+    value: 'existingAccount',
+    short: 'Existing account'
+  },
+  new inquirer.Separator(),
+  {
+    name: 'Exit',
+    value: 'exit'
+  }
+]
+
+const actionQuestion = {
+  type: 'list',
+  name: 'action',
+  message: 'What do you want to do?',
+  choices
+}
+
+const questions = [actionQuestion]
```

Comment:
YAGNI, no need for this if there is only one question

Reply:
I'd thought it to be more expressive this way. How would you do it instead? maybe using `inquirer.prompt([actionQuestion])` below?

Reply:
Leaving it as is for now. The array is not for "future growth support" but for expresiveness, since the `inquirer.prompt(_)` expects an array parameter. The other way of declaring the array in place doesn't seem to be much better?